### ISSUE TITLE
How to install using `--user` option?

### ISSUE THREAD

**Issue by jflorian** [msiemens/PyGitUp#89]
I prefer an install with `pip install --user git-up` to ensure I don't muck up system packages, but then I get the confusing error:

`The script git-up is installed in '/home/00/d13677/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.`

Now what's really odd is that this directory is on my PATH.  In fact, it's the first entry.  I dug a bit and found `GIT_EXEC_PATH determines where Git looks for its sub-programs (like git-commit, git-diff, and others). You can check the current setting by running git --exec-path.` from [the Git docs](https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables).  I got the current value and then used `export GIT_EXEC_PATH="/home/00/d13677/.local/bin;/usr/libexec/git-core"`.  Much to my surprise that didn't help a bit.  If I relocate the `git-up` to another part of my PATH, say `/usr/local/bin` it will work.

Any ideas?

**Comment by msiemens** [comment]
I dont' have any clue as to why this happens either 🙁 Does running `git-up` from your shell work if the `git-up` is located in `/home/.../.local/bin`?

**Comment by jflorian** [comment]
I just stumbled onto the magic, I believe.

I was confused how Pip could be so wrong about my `PATH` and being Python I wasn't afraid to find the code and scrutinize it.  I see no use of `os.path.expanduser()` in `message_about_scripts_not_on_PATH()` of `wheel.py`.  I checked my `PATH` and sure enough I had this particular entry as `~/.local/bin`.  I changed this to `/home/jflorian/.local/bin` and reinstalled PyGitUp.  This time the warning was gone with Pip and `git up` is now happy too.

IMHO both Git and Pip are buggy in that they don't expand `~` in `PATH`.  Maybe that's not conventional as I noticed all the normal default `PATH` entries that I *didn't* put there myself were already expanded ... like whomever did the OS defaults knew better.
